R/tuneParams.R

```diff
@@ -127,3 +127,21 @@ tuneParams = function(learner, task, resampling, measures, par.set, control, sho
 }
 
 
+#' @title Get the optimization path of a tuning result.
+#'
+#' @description
+#' Returns the opt.path from a [\code{\link{tuneParams}}] object.
+#' @param TuneResult [\code{\link{tuneParams}}] \cr
+#'   A tuning result of the [\code{\link{tuneParams}}] function.
```

Comment:
tuneParams -> `TuneResult`

Reply:
Are you sure about this one? I just want to show the user in line 135, that the tuning result is created by the tuneParams function.

Reply:
You are right L:135 it's the result of tuneParams. But in L:134 it has to be TuneResult because that is the class of the object